definitions/3.0.0/tag.json

```diff
@@ -1,15 +1,18 @@
 {
   "type": "object",
+  "description": "Allows adding meta data to a single tag.",
```

Comment:
> Allows adding meta data to a single tag

I don't understand this sentence. Do you mean adding metadata to a single object?

Reply:
Yea, i mean metadata 😄